orttraining/orttraining/python/training/ortmodule/_fallback.py

```diff
@@ -171,7 +172,7 @@ def fallback(self, log_level: _logger.LogLevel, *inputs, **kwargs):
 
         assert self.is_pending(), "`fallback` can only be called when there is a pending fallback"
 
-        if log_level <= _logger.LogLevel.WARNING:
+        if log_level <= _logger.LogLevel.WARNING and not self._raised_ortmodule_onnx_model_exception:
             warnings.warn(
                 (
                     f"Fallback to PyTorch due to exception {type(self._exception)} was triggered. "
```

Comment:
```suggestion
                    "Fallback to PyTorch due to exception %s was triggered. "
```

Reply:
Etc. So that interpolation will not happen when logging level is higher than warning, and thus the conditional is not needed